agent/acs/handler/attach_eni_handler_test.go

```diff
@@ -28,9 +28,9 @@ import (
 )
 
 const (
-	eniMessageId = "123"
-	randomMAC    = "00:0a:95:9d:68:16"
-	waitTimeout  = 10
+	eniMessageId      = "123"
+	randomMAC         = "00:0a:95:9d:68:16"
+	waitTimeoutMillis = 10
```

Comment:
I still don't get it. It's not a unit of time. It only gets converted to a unit of time when multiplied with `time.Millisecond`